contracts/switchboard/native/NativeSwitchboardBase.sol

```diff
@@ -255,41 +247,48 @@ abstract contract NativeSwitchboardBase is ISwitchboard, AccessControl {
         override
         returns (uint256 switchboardFee_, uint256 verificationFee_)
     {
-        return _calculateMinFees(dstChainSlug_);
-    }
-
-    /**
-     * @notice Calculates the minimum switchboard and verification fees required to relay a packet to the destination chain.
-     * @param dstChainSlug_ representing the destination chain identifier.
-     * @return switchboardFee_ representing the minimum switchboard fee required to relay the packet.
-     * @return verificationFee_ representing the minimum verification fee required to relay the packet.
-     */
-    function _calculateMinFees(
-        uint32 dstChainSlug_
-    )
-        internal
-        view
-        returns (uint256 switchboardFee_, uint256 verificationFee_)
-    {
-        (uint256 sourceGasPrice, uint256 dstRelativeGasPrice) = gasPriceOracle__
-            .getGasPrices(dstChainSlug_);
-
-        switchboardFee_ =
-            _getMinSwitchboardFees(
-                dstChainSlug_,
-                dstRelativeGasPrice,
-                sourceGasPrice
-            ) /
-            maxPacketSize;
-
-        verificationFee_ = executionOverhead * dstRelativeGasPrice;
+        return (
+            fees[dstChainSlug_].switchboardFees,
```

Comment:
both fees can be independent of dstChainSlug as dest is fixed in natives

Reply:
dstChainSlug is optional, wont be part of signature schema
this is honor the function signature in `ISwitchboard.sol`